test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua

```diff
@@ -0,0 +1,60 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] [External UCS] SDL informs HMI about <externalConsentStatus> via GetListOfPermissions response
+-- [HMI API] GetListOfPermissions request/response
+-- [HMI API] ExternalConsentStatus struct & EntityStatus enum
+-- When onAppPermission Changed should be sent
```

Comment:
@maria-mih, please put comment like: 
"Clarification: When onAppPermission Changed should be sent"